include/deal.II/differentiation/sd/symengine_tensor_operations.h

```diff
@@ -849,7 +938,113 @@ namespace Differentiation
     }
 
 
-    /* ---------------- Symbolic substitution map creation --------------*/
+    /* ---------------- Symbol map creation and manipulation --------------*/
+
+
+    namespace internal
+    {
+      template <typename ValueType,
+                int rank,
+                int dim,
+                template <int, int, typename> class TensorType>
+      void
+      tensor_set_value_in_symbol_map(
```

Comment:
I guess you have that naming in the existing functions in this file already. If not, have you considered `set_tensor_value_...`?

Reply:
Done. I guess I'll have to change the others at a later stage.